packages/types-dev/index.d.ts

```diff
@@ -236,7 +236,7 @@ declare global {
             message: MessagePayload;
             /** ID of this callback */
             id: number;
-            // ???
+            /** Is it request or response? */
```

Comment:
Is it a question or does this flag allow to distinguish it?

Reply:
It is a description. And this flag allows to distinguish it.

Reply:
Can we then clarify this please, that ack value of true means request and false response or whatever it is, because I still don't get it from the description.^^

Reply:
The comment was extended